src/SeleniumLibrary/keywords/javascript.py

```diff
@@ -42,19 +43,31 @@ def execute_javascript(self, *code):
         This keyword returns whatever the executed JavaScript code returns.
         Return values are converted to the appropriate Python types.
 
+        Starting from SeleniumLibrary 3.2 it is possible to provide JavaScript
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_remote/selenium.webdriver.remote.webdriver.html#selenium.webdriver.remote.webdriver.WebDriver.execute_script|
+        arguments] as part of ``code`` argument. The JavaScript code and
+        arguments must be separated with `JAVASCRIPT` and `ARGUMENTS` markers
+        and must used exactly with this format. If the Javascript code is
+        first, then the `JAVASCRIPT` marker is optional. The order of
+        `JAVASCRIPT` and `ARGUMENTS` markers can swapped, but if `ARGUMENTS`
+        is first marker, then `JAVASCRIPT` marker is mandatory. It is only
+        allowed to use `JAVASCRIPT` and `ARGUMENTS` markers only one time in the
+        ``code`` argument.
+
         Examples:
         | `Execute JavaScript` | window.myFunc('arg1', 'arg2') |
         | `Execute JavaScript` | ${CURDIR}/js_to_execute.js    |
-        | ${sum} =             | `Execute JavaScript` | return 1 + 1; |
-        | `Should Be Equal`    | ${sum}               | ${2}          |
+        | `Execute JavaScript` | alert(arguments[0]); | ARGUMENTS | 123 |
+        | `Execute JavaScript` | ARGUMENTS | 123 | JAVASCRIPT | alert(arguments[0]); |
         """
-        js = self._get_javascript_to_execute(code)
-        self.info("Executing JavaScript:\n%s" % js)
-        return self.driver.execute_script(js)
+        js_code, js_args = self._get_javascript_to_execute(code)
+        self.info('Executing JavaScript:\n%s\nBy using argument(s):\n"%s"'
+                  % (js_code, ', '.join(js_args)))
```

Comment:
Logging could be enhanced a little:
- Better not to mention anything about arguments if there are none. Current `""` looks odd.
- Instead of `"argument(s):"` it's possible to use `"argument%s:" % robot.utils.plural_or_not(js_args)` to get the trailing `s` when needed.
- Quotes around arguments like `"first, second"` look a bit strange. I'd consider omitting them altogether or alternatively quoting each argument separately. In the latter case `robot.utils.seq2str` or `seq2str2` might be worth a look. Just logging `js_args` as-is would be fine otherwise as well, but the `u` prefixes with Unicode strings inside a list in Python 2 are annoying.

Reply:
I did not know that such utils did exist. I definitely will take a look.